dev-client/locales/po/en.po

```diff
@@ -1018,6 +1021,9 @@ msgstr "Next"
 msgid "general##proceed"
 msgstr "Proceed"
 
+msgid "general##offline_title"
```

Comment:
Question: Where did this come from? I don't see the corresponding string in en.json
Or should the changes to this file just be dropped per your slack message?